src/fileformats/ocd_file_export.cpp

```diff
@@ -208,6 +221,52 @@ std::unique_ptr<PointSymbol> postProcessFillPattern(const AreaSymbol::FillPatter
 			point_symbol->addElement(0, object, primary_element);
 		}
 	}
+	
+	// Add and move remaining patterns...
+	using std::begin; using std::end;
+	auto first = begin(patterns) + 1;
+	auto last = end(patterns);
+	
+	// ... but skip second pattern if structure is shifted rows.
+	if (structure_mode == Ocd::StructureShiftedRows)
+	{
+		Q_ASSERT(patterns.size() == 2);
+		if (first != last)
+			++first;
+	}
+	
+	auto i = point_symbol->getNumElements(); 
+	for (auto it = first; it != last; ++it)
+	{
+		auto const* pattern = *it;
+		auto const* pattern_symbol = pattern->point;
+		if ((pattern_symbol->getInnerRadius() > 0 && pattern_symbol->getInnerColor())
+		    || (pattern_symbol->getOuterWidth() > 0 && pattern_symbol->getOuterColor()))
+		{
+			auto* primary_element = new PointSymbol();
+			primary_element->setInnerRadius(pattern_symbol->getInnerRadius());
+			primary_element->setInnerColor(pattern_symbol->getInnerColor());
```

Comment:
The code structure is very similar to the above lines. It might be worth documenting why the earlier code calls `point_symbol->setInnerColor(nullptr)` while this instance does not.

Reply:
Not sure if this is the right place to add documentation: 
- The earlier code *moves* things from `point_symbol` to an element of `point_symbol` (in dealing with the *first* pattern). 
- The code here *clones* things from another point symbol to `point_symbol` (in dealing with *another* pattern).

Reply:
Moved the earlier `point_symbol->setInnerColor(nullptr)` behind another comment.